src/index.html

```diff
@@ -17,11 +17,361 @@
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="card__review">
+                <p>Reviews: 5</p>
+              </div>
+            </div>
+
+            <div class="card__price">
+              <p>Price:</p>
+              <span>$2,199</span>
+            </div>
+
+            <div class="card__btn">
+              <button>BUY</button>
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card__wrap">
+            <div class="card__img">
+              <img
+                src="./images/imac.jpeg"
+                alt="Image of imac"
+              />
+            </div>
+
+            <div class="card__title">
+              <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+              <p>Product code: 195434</p>
+            </div>
+
+            <div class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="card__review">
+                <p>Reviews: 5</p>
+              </div>
+            </div>
+
+            <div class="card__price">
+              <p>Price:</p>
+              <span>$2,199</span>
+            </div>
+
+            <div class="card__btn">
+              <button>BUY</button>
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card__wrap">
+            <div class="card__img">
+              <img
+                src="./images/imac.jpeg"
+                alt="Image of imac"
+              />
+            </div>
+
+            <div class="card__title">
+              <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+              <p>Product code: 195434</p>
+            </div>
+
+            <div class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="card__review">
+                <p>Reviews: 5</p>
+              </div>
+            </div>
+
+            <div class="card__price">
+              <p>Price:</p>
+              <span>$2,199</span>
+            </div>
+
+            <div class="card__btn">
+              <button>BUY</button>
+            </div>
+          </div>
+        </div>
```

Comment:
The product cards are repeated multiple times with identical content. If this is intentional for testing purposes, it's fine. However, if the content is meant to be dynamic, consider using a template or a JavaScript framework to generate these cards dynamically to avoid redundancy and improve maintainability.